aws/ami/files/scylla_install_ami

```diff
@@ -121,3 +121,8 @@ if __name__ == '__main__':
             f.write(u'exclude=kernel kernel-devel')
     elif platform.machine() == 'aarch64':
         run('yum -y --enablerepo=centos-kernel update kernel')
+
+    run('yum install -y yum-utils')
+    packages = subprocess.check_output('repoquery --requires --resolve --recursive scylla', stderr=subprocess.STDOUT, shell=True)
```

Comment:
We could also do `rpm -qa`, but maybe you need this repoquery for some reason. I can't tell because I don't know what the requirements are.

Reply:
Currently we generating recursive dependency list, not all installed packages list, so output of `rpm -qa` and `repoquery --requires --resolve --recursive scylla' are different.
However, I'm not really sure which one is correct, since our motivation is to align installed package between nodes, current way may cause package version difference out side of the dependency list.

@tzach any idea?